aol-htb.js

```diff
@@ -131,12 +149,46 @@ function AolHtb(configs) {
             url += parameter + '=' + requestParams[parameter] + ';';
         }
 
-        return {
-            url: url,
-            callbackId: requestId
-        };
+		return {
+		    url: url,
+			callbackId: requestId
+		};
     }
 
+	function __generateOneMobileRequest(xSlot) {
+        var protocol = Browser.getProtocol();
+		var baseUrl = protocol + endpointsUrls.oneMobile.get;
+		var requestId = '_' + System.generateUniqueId();
+
+		var requestParams = {
+			cmd: 'bid',
+			dcn: xSlot.dcn,
+            pos: xSlot.pos
+		};
+
+		if (protocol === 'https://') {
```

Comment:
The protocol should be matched with `http:` instead.

Reply:
You probably mean with "https:", right?